src/App.jsx

```diff
@@ -13,6 +13,11 @@ import Calendar from './pages/Calendar'
 import About from './pages/About'
 import ContactUs from './pages/ContactUs'
 import EditEntity from './pages/EditEntity'
+import adminsignin from './admin-pages/Signin'
+import AddAdmin from './admin-pages/AddAdmin'
```

Comment:
do you need this in the `App.jsx` ? my guess is no

Reply:
no, sorry this was for testing. I will make sure to delete things that I add for testing